src/content/en/updates/2017/09/picture-in-picture.md

```diff
@@ -0,0 +1,93 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2017-09-07 #}
+{# wf_published_on: 2017-09-07 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Blink>Media #}
+
+# Picture In Picture (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Since April 2017, Chrome for [Android O supports Picture In Picture]. It allows
+users to play a `<video>` element in a pinned window that isn't blocked by other
+windows, so that they can watch while doing other things.
+
+Here's how it works: Open Chrome, go to a website that contains a video and
+play it fullscreen. From there, press the Home button to go to your Android
+Home Screen and the playing video will automatically transition in Picture In
+Picture. That's all! Pretty cool right?
+
+<figure>
+  <img src="https://dummyimage.com/1798x854/999999/ffffff&text=Chrome+PiP+screenshot"
+       alt="Chrome PiP screenshot">
+  <figcaption>
+    <b>Figure 1.</b>
+    TODO: Chrome PiP screenshot
+  </figcaption>
+</figure>
+
+It is but... what about desktop? What if the website want to control that
+experience?
+
+The good news is that a [Picture In Picture Web API] specification is being
+drafted as we speak. This spec aims to allow websites to initiate and control
+this behavior by exposing the following sets of properties to the API:
+
+- Notify the website when it enters and leave Picture in Picture mode.
+- Allow the website to trigger Picture in Picture via a user gesture on a video element.
+- Allow the website to exit Picture in Picture.
+- Allow the website to check if Picture in Picture can be triggered.
```

Comment:
On the four points above, can you make it clear that it's the video going PIP? you say "the website" and "it" which can put the wrong expectations.

Reply:
Done.